chainer/distributions/normal.py

```diff
@@ -0,0 +1,239 @@
+import chainer
+from chainer.backends import cuda
+from chainer import Distribution
+from chainer.functions.array import broadcast
+from chainer.functions.array import expand_dims
+from chainer.functions.array import repeat
+from chainer.functions.math import erf
+from chainer.functions.math import erfinv
+from chainer.functions.math import exponential
+import numpy
+
+
+class Normal(Distribution):
+
+    """Normal Distribution.
```

Comment:
Better to write how the normal distribution is parameterized (e.g., write down the density function using the same variable names as the argument documentation: mu and sigma).